Readme.md

```diff
@@ -63,6 +63,25 @@ a core value of our community. Consider the points of [the opposition][], but
 be a traditionalist when it comes to abusing error correction mechanisms for
 cheap syntactic pleasures.
 
+### Do not append semicolons after function unless declared with var
```

Comment:
Thumbs up. also, i think this needs to go under the 2 links below (they relate to above paragraph)